### DISCUSSION TITLE
make launch off screen mode?

### DISCUSSION THREAD

**Discussion by E-Chow** [carla-simulator/carla#8235] [Q&A] [answered]
Can I do make launch with render off screen option? So far I found `./CarlaUE4.sh -RenderOffScreen` in the document, but I am building carla with make launch, I don't see this option in any of the make files. Can someone tell me how to run off screen mode with make launch?

**Comment by asalarpour** [comment]
make launch ARGS='--editor-flags="-opengl -RenderOffScreen --carla-rpc-port=2000"'

**Reply by E-Chow** [reply, marked as answer]
Thanks for the reply. This isn't the solution, however, you inspired me to look further into the script of CarlaUE4.sh and Build tools, and I found out a way to modify BuildCarlaUE4.sh to pass these argument into Editor. Here's the way to do it, you can change RHI="-vulkan" to RHI="-RenderOffScreen" then do make launch.
